bitcoincore/src/main/kotlin/io/horizontalsystems/bitcoincore/managers/BlockchainComApi.kt

```diff
@@ -28,16 +31,24 @@ class BlockchainComApi(transactionApiUrl: String, blocksApiUrl: String) : IIniti
                 )
             }
 
+            var blockHeight : Int = -1
+            if (!transaction["block_height"].isNull) {
+                blockHeight = transaction["block_height"].asInt()
+            }
```

Comment:
Setting `blockHeight` to `-1` so that we know if we should include it as part of the transactions that we fetch blocks for.